src/components/product/ProductImages.tsx

```diff
@@ -78,7 +78,7 @@ const getProductImageError = (
             typeof Object.values(errors.productImages)?.[0] === "string"
           ? Object.values(errors.productImages)?.[0]
           : null;
-  return error;
+  return String(error);
```

Comment:
if error is null then this will be 'null'
```suggestion
  return error? String(error) : error;
```